<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>AddFundDialog</class>
 <widget class="QDialog" name="AddFundDialog">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>340</width>
    <height>169</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Dialog</string>
  </property>
  <layout class="QGridLayout" name="gridLayout">
   <property name="bottomMargin">
    <number>9</number>
   </property>
   <item row="2" column="2">
    <widget class="QPushButton" name="cancelBtn">
     <property name="focusPolicy">
      <enum>Qt::NoFocus</enum>
     </property>
     <property name="text">
      <string>取消</string>
     </property>
    </widget>
   </item>
   <item row="2" column="1">
    <widget class="QPushButton" name="saveBtn">
     <property name="focusPolicy">
      <enum>Qt::TabFocus</enum>
     </property>
     <property name="text">
      <string>保存</string>
     </property>
     <property name="shortcut">
      <string>Return</string>
     </property>
    </widget>
   </item>
   <item row="2" column="0">
    <spacer name="horizontalSpacer">
     <property name="orientation">
      <enum>Qt::Horizontal</enum>
     </property>
     <property name="sizeHint" stdset="0">
      <size>
       <width>40</width>
       <height>20</height>
      </size>
     </property>
    </spacer>
   </item>
   <item row="0" column="0" colspan="3">
    <layout class="QGridLayout" name="gridLayout_2">
     <property name="verticalSpacing">
      <number>15</number>
     </property>
     <item row="0" column="0">
      <widget class="QLabel" name="Label">
       <property name="text">
        <string>基金代码</string>
       </property>
      </widget>
     </item>
     <item row="0" column="1">
      <widget class="QLineEdit" name="fundCode"/>
     </item>
     <item row="1" column="0">
      <widget class="QLabel" name="Label_2">
       <property name="text">
        <string>持仓成本</string>
       </property>
      </widget>
     </item>
     <item row="1" column="1">
      <widget class="QLineEdit" name="fundCost"/>
     </item>
     <item row="2" column="0">
      <widget class="QLabel" name="Label_3">
       <property name="text">
        <string>持有份额</string>
       </property>
      </widget>
     </item>
     <item row="2" column="1">
      <widget class="QLineEdit" name="fundUnits"/>
     </item>
    </layout>
   </item>
  </layout>
 </widget>
 <tabstops>
  <tabstop>fundCode</tabstop>
  <tabstop>fundCost</tabstop>
  <tabstop>fundUnits</tabstop>
  <tabstop>saveBtn</tabstop>
  <tabstop>cancelBtn</tabstop>
 </tabstops>
 <resources/>
 <connections>
  <connection>
   <sender>saveBtn</sender>
   <signal>clicked()</signal>
   <receiver>AddFundDialog</receiver>
   <slot>accept()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>198</x>
     <y>145</y>
    </hint>
    <hint type="destinationlabel">
     <x>151</x>
     <y>138</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>cancelBtn</sender>
   <signal>clicked()</signal>
   <receiver>AddFundDialog</receiver>
   <slot>reject()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>292</x>
     <y>147</y>
    </hint>
    <hint type="destinationlabel">
     <x>336</x>
     <y>132</y>
    </hint>
   </hints>
  </connection>
 </connections>
</ui>
